core/domain/stats_domain.py

```diff
@@ -466,30 +466,99 @@ def validate(self):
                 self.classification_categorization)
 
 
+class AnswerOccurrence(object):
+    """Domain object that represents a specific answer that occurred some number
+    of times.
+    """
+    def __init__(self, answer, frequency):
+        """Initialize domain object for answer occurrences."""
+        self.answer = answer
+        self.frequency = frequency
+
+    def to_dict(self):
+        return {
+            'answer': self.answer,
+            'frequency': self.frequency
+        }
+
+
+class AnswerFrequencyList(object):
+    """Domain object that represents a list of AnswerOccurrences."""
+    def __init__(self, answer_occurrences=None):
+        """Initialize domain object for answer frequency list for a given list
+        of AnswerOccurrence objects (default is empty list).
+        """
+        self.answer_occurrences = (
+            answer_occurrences if answer_occurrences else [])
+
+    def add_answer(self, answer_occurrence):
+        """Adds a new AnswerOccurrence object."""
+        self.answer_occurrences.append(answer_occurrence)
+
+    def to_dict(self):
```

Comment:
Doesn't look like a dict to me...

Reply:
Was tricky to name this since both objects need to have the same ``to_x`` method. Thoughts on ``to_raw_type``?

Reply:
to_raw_type seems reasonable for what this is doing. Though the polymorphism seems a bit weird to me; see comment below.

Reply:
Unless I'm mistaken, that's unavoidable here. Your comment seemed to be based on the incoming type--do you have another suggestion for how to approach this without a type field? Both approaches I suggested here require the type field; I couldn't think of another approach.